pkg/images/enricher/enricher_impl.go

```diff
@@ -655,6 +658,10 @@ func (e *enricherImpl) checkRegistryForImage(image *storage.Image) error {
 	return nil
 }
 
+func isOpenshiftGlobalRegistry(source *storage.ImageIntegration_Source) bool {
```

Comment:
```suggestion
func isOpenshiftGlobalPullSecret(source *storage.ImageIntegration_Source) bool {
```
the pull secret has nothing to do with an OpenShift registry, at least not necessarily so